test/command_callback/test_llc_command_callback.vader

```diff
@@ -0,0 +1,29 @@
+Before:
+  Save g:ale_llvm_llc_executable
+
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+
+  runtime ale_linters/llvm/llc.vim
+
+After:
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+  Restore
+
+Execute(llc command is customizable):
+  let cmd = ale_linters#llvm#llc#GetCommand(bufnr(''))
```

Comment:
Test the return value of `GetExecutable` too, and ensure it is not escaped.

Reply:
I'll do.